src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -47,12 +61,6 @@ public function persist(array $objects)
      */
     public function find($class, $id)
     {
-        $entity = $this->om->find($class, $id);
-
-        if (!$entity) {
```

Comment:
Yep. I don't personally have a string opinion about wether an exception should be thrown or null returned, but it's better to stay consistent